components/Habit.js

```diff
@@ -36,35 +49,23 @@ export default function Habit(props) {
     <>
       { props.isEditable ?
         (
-          <div className='card' style={{ backgroundColor: props.color}} onClick={styleCheckmark}>
+          <div className='card' style={{ backgroundColor: props.color}} onClick={completeHabit}>
             <div className="topLayer">
               <div className='emojiContainer'>
-                  { emoji ? 
-                    (
-                      <>
-                        <div className='emoji' onClick={handleEmojiClick}>{ emoji }</div>
-                        <div className='shadow'></div>
-                      </>
-                    )
-                    :
-                    (
-                      <div className='addEmoji'>
-                        {
-                          emoji === "" ? <p className="addEmojiText">☻ Add icon</p> : <div className="emoji">{ emoji }</div>
-                        }
-                        <div className='shadow'></div>
-                      </div>
-                    )
-                  }
+                <div className='emoji' onClick={emojiClickHandler}>{ emoji }</div>
+                <div className='shadow'></div>
               </div>
               <label className='container'>
                 <div className="circle">
                   <img src={checkmark} alt="SVG as an image"/>
                 </div>
               </label>
             </div> 
-            <div onClick={(e) => e.stopPropagation()}>
-            <InlineEdit editingValue={editingValue} setEditingValue={setEditingValue}/>
+            <div onClick={(e) => e.stopPropagation()} className='inline'>
+              <InlineEdit editingValue={editingValue} setEditingValue={setEditingValue} textDecoration={textDecoration}/>
+            </div>
+            <div className='trashcan'>
+              <img src='/trash.svg' onClick={deleteHandler}/>
```

Comment:
Can we move the trashcan lower on the card?
<img width="443" alt="CleanShot 2023-01-09 at 19 52 58@2x" src="https://user-images.githubusercontent.com/4623985/211437558-95978d33-15a4-4337-b4c9-50bd6d908fd5.png">

Reply:
Done